docs/en/observability/universal-profiling.asciidoc

```diff
@@ -0,0 +1,226 @@
+[[universal-profiling]]
+= Universal Profiling
+
+beta::[]
+
+Elastic Universal Profiling is a continuous profiling tool running on top of the {stack}.
+
+For a quick overview of Universal Profiling, see the https://www.elastic.co/observability/ebpf-continuous-code-profiling[Universal Profiling product page]
+
+Here you'll find information on:
+
+* inspecting data through <<profiling-stacktraces-intro,stacktraces>>, <<profiling-flamegraphs-intro,flamegraphs>>, and <<profiling-functions-intro,functions>>
+* <<profiling-filtering-intro,Filtering>> your data
+* comparing time ranges through <<profiling-differential-views-intro,differential views>>
+
+[discrete]
+[[profiling-inspecting-data-in-kibana]]
+== Inspecting data in {kib}
+
+You can find Universal Profiling in the *Observability* navigation menu. Clicking *Stacktraces* under *Universal Profiling* opens the <<profiling-stacktraces-intro,stacktraces view>>.
+
+NOTE: Universal Profiling currently only supports CPU profiling through stack sampling.
+
+From the *Stacktraces* view, you get an overview of all of your data. You can also use filtering queries in the search bar to slice your data into more detailed sections of your fleet.
+Time-based filters and property filters allow you to inspect portions of data and drill-down into how much CPU various parts of your infrastructure consume over time.
+
+See <<profiling-filtering-intro,Filtering>> for more on slicing data and <<profiling-differential-views-intro,Differential views>> for more on comparing two time ranges to detect performance improvements or regressions.
+
+[discrete]
+[[profiling-debug-symbols-intro]]
+=== Debug symbols
+
+Your stacktrace can be:
+
+* symbolized, showing the full source code's filename and line number
+* partially symbolized
+* not symbolized
+
+In the following screenshot, you can see that unsymbolized frames _do not_ show a filename and line number, but a hexadecimal number such as `0x80d2f4`
+or `<unsymbolized>`.
+
+Adding symbols for unsymbolized frames is currently a manual operation. See *LINK NEEDED*.

```

Comment:
Seems there is a missing link (`LINK NEEDED`)? This should probably point to `profiling-add-symbols.asciidoc`.